lib/rspec/core/metadata.rb

```diff
@@ -47,7 +47,9 @@ def self.relative_path(line)
         return nil if line == '-e:1'.freeze
         line
       rescue SecurityError
+        # :nocov:
         nil
+        # :nocov:
       end
```

Comment:
RuboCop warns of the empty `rescue`. It can be suppressed with:

```ruby
      rescue SecurityError # rubocop:disable Lint/HandleExceptions
      end
```

but I think the explicit `nil` with `:nocov` is simpler.